docs/website/content/v0.3/en/guides/metal/index.md

```diff
@@ -34,6 +34,10 @@ Talos also enforces some minimum requirements from the KSPP (kernel self-protect
 - `slab_nomerge`
 - `pti=on`
 
+You may experience long boot times due to low entropy.
+In newer versions of Linux, the initialization of the Cryptographically (Secure) Random Number Generator (CRNG) will block until it is properly seeded from the entropy pool at boot time.
```

Comment:
Do we know what constitutes as a newer version? Might be worth doing something like (kernel >= 5.0.0)

Reply:
Not sure, but since we control the kernel version I think we can skip since we know that the bundled kernel requires it. If needing to know which version becomes an issue, then we can add.